inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/CreateCommand.java

```diff
@@ -51,26 +51,38 @@ private static class CreateGroup extends AbstractCommandRunner {
 
         @Parameter(names = {"-f", "--file"},
                 converter = FileConverter.class,
-                required = true,
                 description = "json file")
         private File file;
 
+        //TODO:add gdbc connector and others
```

Comment:
Suggest leave a blank between start of comment // and _text_ which is the case in all other inlong source files.